### ISSUE TITLE
Does not copy index.html

### ISSUE THREAD

**Issue by coderpradp** [KieferSivitz/vue-gh-pages#17]
I used webpack-simple template. When I deploy, it throws me with this error:
```
Build Complete.
Preparing files for github pages
{ Error: ENOENT: no such file or directory, open 'E:\{path}\docs\index.html'
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'E:\\{path}\\docs\\index.html' }
```

**Comment by Adamkillander96** [comment]
I get the same error.

```
Build Complete.
Preparing files for github pages
{ Error: ENOENT: no such file or directory, open 'C:\{path}\docs\index.html'
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'C:\\{path}\\docs\\index.html' }
```

Anything more I should provide to help?

**Comment by skayablars** [comment]
It's happening to me too if it's any help I detail my configuration.

OS: Windows 10
NVM: (Windows one) 1.1.6
NPM: 5.6.0

Build Complete.
Preparing files for github pages
{ Error: ENOENT: no such file or directory, open 'C:\Users\larizza\Downloads\vue-course\s18\docs\index.html'
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'C:\\Users\\larizza\\Downloads\\vue-course\\s18\\docs\\index.html' }

**Comment by KieferSivitz** [comment]
I was just able to reproduce this. Will be looking into this and the webpack-simple problem(s) this weekend

**Comment by KieferSivitz** [comment]
This _should_  be resolved now (as version 1.12.0) if it is not open a new issue and I'll look into it
